client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -699,7 +699,9 @@ class CalypsoifyIframe extends Component<
 						/* eslint-enable jsx-a11y/iframe-has-title */
 					) }
 				</div>
-				<EditorMediaModal
+				<AsyncLoad
+					require="post-editor/editor-media-modal"
```

Comment:
Idea for a follow-up: move the component out of `post-editor` to a better place, like `blocks/editor-media-modal`.

It won't be long before we merge a PR where we simply delete the entire `post-editor` folder. Let's move the surviving code out of it in the meantime.

Reply:
Cool idea! It will make removing `post-editor` one step closer once we decide to fully get rid of it.

Adding to my list of ideas to revisit once we ship this one.